### PR TITLE
#37 - Add required entries into iOS plist file

### PR DESCRIPTION
**Pull request by mark-veenstra**
Solves #37

### CODE REVIEW

File: plugin.xml [hunk 1]
```diff
@@ -76,5 +76,13 @@
       <config-file target="config.xml" parent="/*">
         <access origin="*" />
       </config-file>
+
+      <!-- add required entries into plist file -->
+      <edit-config target="NSCameraUsageDescription" file="*-Info.plist" mode="merge" overwrite="true">
```
Comment (msach22) [inline plugin.xml]: @mark-veenstra do we have to set `overwrite` to true? iOS allows you to customize the messages so what if the developer has already set these descriptions?

File: plugin.xml [hunk 2]
```diff
@@ -76,5 +76,13 @@
...
+        <string>The camera is required to publish video</string>
+      </edit-config>
+      <edit-config target="NSMicrophoneUsageDescription" file="*-Info.plist" mode="merge" overwrite="true" >
+        <string>The microphone is required to publish audio</string>
```
Comment (msach22) [inline plugin.xml]: @mark-veenstra same as above

File: plugin.xml (hunk 1 above)
Comment (mark-veenstra) [reply plugin.xml]: Yeah that is probably a bit aggresive to set. I removed the `overwrite`

File: plugin.xml (hunk 2 above)
Comment (mark-veenstra) [reply plugin.xml]: same

**Comment by mark-veenstra** [comment]
@msach22 Can you cherry pick this one out? Seems to give more issues than it solves after some tests!

**Comment by msach22** [comment]
@mark-veenstra I just manually reverted the contents from the commit. thanks for letting me know

**Pull request merged**